packages/design-system-tokens/src/themes/healthcare-component.ts

```diff
@@ -239,10 +239,10 @@ export const components: AnyTokenValues = {
     '__border-color--disabled':                   'transparent',
     '__border-color--hover':                      'transparent',
     '__border-color':                             'transparent',
-    '__color--active':                            t.color['secondary'],
+    '__color--active':                            t.color['gray-lighter'],
     '__color--disabled':                          t.color['gray-dark'],
-    '__color--hover':                             t.color['secondary-light'],
-    '__color':                                    t.color['secondary-lighter'],
+    '__color--hover':                             t.color['gray-lightest'],
+    '__color':                                    t.color['white'],
```

Comment:
Like I mentioned in the description, I simplified the unused healthcare button styles by copying the `main` to `alternate` styles. Below is a screenshot of the changes:

<details>
  <summary>Before/After screenshot</summary>
  
  ![healthcare-buttons-before-after](https://user-images.githubusercontent.com/7595652/218901138-7ee85702-0bda-42a6-bd77-636970d1e8e4.png)
</details>